src/web3/networkConfig.ts

```diff
@@ -180,11 +184,29 @@ const CELO_EXPLORER_BASE_ADDRESS_URL_MAINNET = `${CELO_EXPLORER_BASE_URL_MAINNET
 const CELO_EXPLORER_BASE_TOKEN_URL_ALFAJORES = `${CELO_EXPLORER_BASE_URL_ALFAJORES}/token/`
 const CELO_EXPLORER_BASE_TOKEN_URL_MAINNET = `${CELO_EXPLORER_BASE_URL_MAINNET}/token/`
```

Comment:
should we remove these and the blockscout related variables in the network config object? i think only NFT details still should go through Blockscout (although even that i'm doubting because of [this](https://valora-app.slack.com/archives/CL7BVQPHB/p1698335014761419?thread_ts=1696001827.200189&cid=CL7BVQPHB) - i recall @MuckT saying that CeloScan does not display NFT metadata as well as Blockscout but it might still be worth using CeloScan instead)

Reply:
I kept these variables only for NFTs, not really sure that switch to CeloScan brings better user experience over Blockscout for NFT users.

As far as I understood the slack discussion you've mentioned, another block explorer is cooked. Maybe it's too early to switch now?

Reply:
yes i'm also not sure what we want to do with the NFTs right now, so let's leave it. it seems like we can remove `celoExplorerBaseTxUrl` and `celoExplorerBaseAddressUrl` though? and also `BLOCKSCOUT_BASE_URL` from config.ts?